src/views/Main/Link/Link.js

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Select from 'react-select';
+import Promise from 'bluebird';
+import { EntityDataModelApi } from 'loom-data';
+import Page from '../../../components/page/Page';
+import buttonStyles from '../../../core/styles/buttons.css';
+import styles from './styles.module.css';
+
+export class Link extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      allEntitySets: [],
+      selectedEntitySetIds: [],
+      availablePropertyTypes: {},
+      links: [],
+      entityTypeIdToEntitySet: {},
+      propertyTypeIdToEntityType: {},
+      newRow: true,
+      editingPropertyType: '',
+      editingEntitySets: [],
+      needsTwoEntitySetsError: false
+    };
+  }
+
+  componentDidMount() {
+    this.loadAllEntitySets();
+  }
+
+  loadAllEntitySets = () => {
+    EntityDataModelApi.getAllEntitySets()
+    .then((allEntitySets) => {
```

Comment:
I think it's good practice to add `.catch()` to the Promise chain to handle any errors in case something goes wrong